intel-retail/automated-self-checkout
# Expand profile-launcher configs to support the generic docker-run script

Add more configuration into profile-launcher (pl) so that it can support the generic docker-run script that Brian is working on.



## PR Checklist



## What are you changing?


## Issue this PR will close

close: #**issue_number**

## Anything the reviewer should know when reviewing this PR?

## Test Instructions if applicable

You can run smoke test and everything should pass:
```console
make run-smoke-tests
```

```text
results of smoke tests recorded in the file smoke_tests_output.log
=== test rtsp://127.0.0.1:8554/camera_0 profile smoke test status code PASSED
=== classification rtsp://127.0.0.1:8554/camera_0 profile smoke test status code PASSED
=== classification rtsp://127.0.0.1:8554/camera_0 profile smoke test pipeline log PASSED
=== grpc_go rtsp://127.0.0.1:8554/camera_0 profile smoke test status code PASSED
=== grpc_go rtsp://127.0.0.1:8554/camera_0 profile smoke test pipeline log PASSED
=== grpc_python rtsp://127.0.0.1:8554/camera_0 profile smoke test status code PASSED
=== grpc_python rtsp://127.0.0.1:8554/camera_0 profile smoke test pipeline log PASSED
=== gst rtsp://127.0.0.1:8554/camera_0 profile smoke test status code PASSED
=== gst rtsp://127.0.0.1:8554/camera_0 profile smoke test pipeline log PASSED
=== instance_segmentation rtsp://127.0.0.1:8554/camera_0 profile smoke test status code PASSED
=== instance_segmentation rtsp://127.0.0.1:8554/camera_0 profile smoke test pipeline log PASSED
=== object_detection rtsp://127.0.0.1:8554/camera_1 profile smoke test status code PASSED
=== object_detection rtsp://127.0.0.1:8554/camera_1 profile smoke test pipeline log PASSED
```
## If the there are associated PRs in other repositories, please link them here (i.e. intel-retail/automated-self-checkout )
